### PR TITLE
Travis-ci: added support for ppc64le for react-router

### PR DESCRIPTION
**Pull request by dthadi3**
I had added ppc64le(Linux on Power) architecture support on Travis-CI in the PR

### CODE REVIEW

File: travis-ymls/react-router.travis.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : react-router
+# Source Repo         : https://github.com/ReactTraining/react-router
+# Travis Job Link     : https://travis-ci.com/github/dthadi3/react-router/builds/210998601
+# Created travis.yml  : No
+# Maintainer          : Devendranath Thadi <devendranath.thadi3@gmail.com>
+#
+# Script License      : MIT License, Version 2 or later
```
Comment (seth-priya) [inline travis-ymls/react-router.travis.yml:9]: Script license should always be Apache 2.0

**Review by seth-priya — CHANGES_REQUESTED** [review]

File: travis-ymls/react-router.travis.yml (hunk 1 above)
Comment (dthadi3) [reply travis-ymls/react-router.travis.yml]: Done.

**Comment by seth-priya** [comment]
@ghatwala can you please review the travis.yml for this one?

File: travis-ymls/react-router.travis.yml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    env: TEST_ENV=source
+    before_script:
+    # react-router CJS build is imported by react-router-config tests,
+    # so we need to build it before running react-router-config tests
+    - cd packages/react-router && BUILD_ENV=cjs yarn build && cd -
+    script: yarn test
+  - env: TEST_ENV=cjs BUILD_ENV=cjs
+    before_script: yarn build
+    script: yarn test
+  - env: TEST_ENV=umd BUILD_ENV=umd
```
Comment (ghatwala) [inline travis-ymls/react-router.travis.yml:28]: Can we add `amd64` arch just for consistency for all 3 envs ?- source , cjs , umd - @dthadi3 , rest all looks good.

Comment (dthadi3) [reply travis-ymls/react-router.travis.yml:30]: Added amd64 arch for all the 3 env's.

**Comment by seth-priya** [comment]
@dthadi3 please squash the commits

**Comment by dthadi3** [comment]
Raised New PR #401

**Pull request closed without merging**